monailabel/tasks/infer/basic_infer.py

```diff
@@ -226,6 +226,11 @@ def inferer(self, data=None) -> Inferer:
         roi_size = data.get("roi_size", self.roi_size) if data else self.roi_size
         sw_batch_size = data.get("sw_batch_size", 1) if data else 1
         sw_overlap = data.get("sw_overlap", 0.25) if data else 0.25
+
+        # Update list of device Ids with "cuda" as it was  loaded with GPU marketing display name; monai core needs id to be "cuda"
+        req = copy.deepcopy(self._config)
+        req.update({"device": device_list()})
```

Comment:
device_list is not an array here?  in that case device = data.get("device") will return the list.. what is the further use-case here?

have you guys verified on multi-gpu? is it all ok?  have u tried running on selected gpu in multi-gpu use-case.. also on all gpus..